calculator-styles.css

```diff
@@ -0,0 +1,123 @@
+* {
+  font-family: sans-serif;
+}
+.calculator {
+  border-color: transparent;
+  border-style: solid;
+  box-sizing: border-box;
```

Comment:
Apply this property to the global selector `*` since it is better to have it there.